addons/mail/static/src/components/autocomplete_input/autocomplete_input.xml

```diff
@@ -2,7 +2,7 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.AutocompleteInput" owl="1">
-        <input class="o_AutocompleteInput" t-on-blur="_onBlur" t-on-keydown="_onKeydown" t-att-placeholder="props.placeholder"/>
+        <input class="o_AutocompleteInput" t-on-blur="_onBlur" t-on-focusin="props.onFocusin" t-on-keydown="_onKeydown" t-att-placeholder="props.placeholder"/>
```

Comment:
onFocusin is optional but not guarded, just to make sure, it's ok to give undefined event handler?

Reply:
That's the only aspect I'm not sure whether OWL appreciates empty `t-on`. Runbot's green and everything seems to work fine so apparently this is fine.